core/src/main/resources/hudson/triggers/TimerTrigger/help-spec.html

```diff
@@ -78,4 +78,31 @@
 # once a day on the 1st and 15th of every month except December
 H H 1,15 1-11 *
 </pre>
+<p>
+  <b>Timezones specification</b>. <br/>
+  Starting from 1.615 Jenkins supports the specification 
+  of different timezones in this field. It allows to natively configure all
```

Comment:
"of *a time zone*" would be clearer as it's only possible to specify one (and it doesn't have to be different from the default).